.github/actions/status-checker/src/status-checker.ts

```diff
@@ -12,13 +12,13 @@ export async function checkStatus(token: string) {
 
   if (prNumber) {
 
-    const { data: pullCommits } = await octokit.rest.pulls.listCommits({
+    const { data: pullCommits } = await octokit.pulls.listCommits({
```

Comment:
@BillWagner The second commit just removes "rest" here. That seems to be the normal way to call this API.